draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -97,8 +97,8 @@ This document uses the terms defined in {{Section 2 of !RFC5681}}.
 
 Irrespective of the current state of a congestion control mechanism, senders of congestion controlled transport protocols:
 
-1. MUST limit the growth of cwnd when FlightSize < cwnd.
-2. SHOULD limit the growth of cwnd with inc(maxFS).
+1. MUST limit the growth of cwnd when FlightSize < cwnd in some way.
+2. SHOULD limit the growth of cwnd when FlightSize < cwnd with inc(maxFS).
```

Comment:
MUST with /some way/ sounds fuzzy.
I guess the nearest could be:
/MUST include a way to limit.../

Reply:
I like that!  I'll try this now and merge.  We should have a first version available soon, I think - we can always fine-tune the -01 draft much more  :-)